src/server.rs

```diff
@@ -440,6 +444,24 @@ impl Server {
                     break;
                 }
 
+                // CommandComplete
+                'C' => {
+                    let full_message = String::from_utf8_lossy(message.as_ref());
+                    let mut it = full_message.split_ascii_whitespace();
+                    let command_tag = it.next().unwrap().trim_end_matches(char::from(0));
```

Comment:
I would be careful with `unwrap()` here, we don't want to panic the thread. Maybe do a `match`?